playbooks/slave_open_ports.yml

```diff
@@ -0,0 +1,11 @@
+  - name: Secure Slave
```

Comment:
I'm fine for merging this with the other, but I thought it was good to keep it independant.